build/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM pryv/base:1.5.1 as build-stage
```

Comment:
why do you use pryv image and not something lighter like node:lts-alpine for building the first image?

Reply:
I was told to do so by Ilia. That we should use this image in every case